Resources/public/templates/dashboard/allcontent-dashboard-block.hbt

```diff
@@ -0,0 +1,36 @@
+<h3 class="ez-allcontent-block-title">All Content</h3>
```

Comment:
well in that case, the module title are clearly an important thing from a structure point of view. One of the aspect of the *hN* structure is to be able to build a table of contents that can be used to guess the page structure. So here, the block title is definitively the second level in the heading hierarchy.